examples/wideresnet/model.py

```diff
@@ -53,7 +54,8 @@ def schedule_model(
     sch = slapo.create_schedule(model, group=group)
     logger.info(f"Scheduling Wide-ResNet with TP={sch.world_size}", ranks=0)
 
-    fuse_conv_bn(sch[prefix], config)
+    if dist.get_world_size() == 1:  # not only counting the TP devices
```

Comment:
Use `sch.world_size` would be our best practice to make use this is for a TP group.
```suggestion
    if sch.world_size == 1:  # not only counting the TP devices
```